subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -207,7 +219,51 @@ void lll_df_conf_cte_rx_enable(uint8_t slot_duration, uint8_t ant_num, uint8_t *
 }
 #endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
 
+/* @brief Function performs common steps for initialization and reset
+ * of Direction Finding LLL module.
+ *
+ * @return	Zero in case of success, other value in case of failure.
```

Comment:
Only use spaces for indentation if anything but a tab comes first
```suggestion
 * @return    Zero in case of success, other value in case of failure.
```